src/teleport.cpp

```diff
@@ -85,45 +91,47 @@ void Teleport::addThing(int32_t, Thing* thing)
 
 	const MagicEffectClasses effect = Item::items[id].magicEffect;
 
-	if (Creature* creature = thing->getCreature()) {
+	if (auto creature = thing->getCreature()) {
```

Comment:
Could this be `const auto`
```suggestion
	if (const auto creature = thing->getCreature()) {
```